### DISCUSSION TITLE
Renew server_"randoms".crt

### DISCUSSION THREAD

**Discussion by pikedhann** [angristan/openvpn-install#840] [Help]
Hi, 

i have an installation of openvpn with the Angristan's script by 3 years

The certified called "server_cnsuinvis***.crt" is expired (not ca.crt) and clients no longer connect

How can I update this certificate?

tnk

**Comment by yta4000** [comment]
https://github.com/angristan/openvpn-install/issues/974#issuecomment-1057339249
